### ISSUE TITLE
I can not scan aztec code

### ISSUE THREAD

**Issue by GruszkaMarcin** [EddyVerbruggen/nativescript-barcodescanner#167]
Hi, I have a problem with aztec code scanning, the scanner has problems with code recognition, in the attachment I attach an example picture

![barcode 1](https://user-images.githubusercontent.com/20757754/47288940-b3cd9300-d5f8-11e8-8c6f-906269a1d063.gif)

**Comment by EddyVerbruggen** [comment]
I don't think this plugin will be enhanced regarding those codes, but you could use [Firebase ML Kit](https://firebase.google.com/docs/ml-kit/read-barcodes) instead via https://github.com/EddyVerbruggen/nativescript-plugin-firebase/blob/master/docs/ML_KIT.md#barcode-scanning

**Comment by alexZaicev** [comment]
Hi Eddy could update the documentation that AZTEC code scanner does not work (in my case, I only tested on Android) :)

**Comment by EddyVerbruggen** [comment]
ok..

**Comment by chrum** [comment]
hello,

I just tested the results of this PR https://github.com/EddyVerbruggen/barcodescanner-lib-aar/pull/23 and it works like a dream.

any chance to accept it?

**Comment by EddyVerbruggen** [comment]
I may have missed that PR, thanks for bringing it to my attention! Can you perhaps share a code that wasn't recognized before and is with that PR applied? If it's too much of a hassle then don't worry about it.

**Comment by chrum** [comment]
this one i think
![image](https://user-images.githubusercontent.com/4775239/59865391-2d38a380-9389-11e9-8c54-27b483c64ee4.png)

**Comment by chrum** [comment]
i can also scan and send you my car id but that is a bit sensitive so maybe on email ;)

**Comment by Tekunoff** [comment]
> hello,
> 
> I just tested the results of this PR [EddyVerbruggen/barcodescanner-lib-aar#23](https://github.com/EddyVerbruggen/barcodescanner-lib-aar/pull/23) and it works like a dream.
> 
> any chance to accept it?

Hello. I've installed EddyVerbruggen/barcode scanner-lib-aa, but didn't succeed to recognize any Aztec codes. Can I ask you about consultation?

**Comment by chrum** [comment]
@Tekunoff 
did you try to build new .aar with the changes from that PR?
if not then you can use the one located at:
https://github.com/dex4er/phonegap-plugin-barcodescanner/tree/master/src/android

what i am doing next is simply replacing
`node_modules/nativescript-barcodescanner/platforms/android/barcodescanner-release-2.1.6.aar`

with this new one

**Comment by Tekunoff** [comment]
> @Tekunoff
> did you try to build new .aar with the changes from that PR?
> if not then you can use the one located at:
> https://github.com/dex4er/phonegap-plugin-barcodescanner/tree/master/src/android
> 
> what i am doing next is simply replacing
> `node_modules/nativescript-barcodescanner/platforms/android/barcodescanner-release-2.1.6.aar`
> 
> with this new one

Thank you for your answer! I'll try that way.

**Comment by Tekunoff** [comment]
@chrum could you recognize this one? http://joxi.ru/L21NjZvHRJ7OJA

**Comment by chrum** [comment]
nope, but i think thats because the quality is rather bad :(

codes in this thread are recognizable after applying changes for sure
